doc/source/ray-core/placement-group.rst

```diff
@@ -146,28 +151,20 @@ Strategy types
 
 Ray currently supports the following placement group strategies:
 
-**STRICT_PACK**
-
-All bundles must be placed into a single node on the cluster.
+**PACK**: All provided bundles are packed onto a single node on a best-effort basis.
 
-**PACK**
+**STRICT_PACK**: All bundles must be placed into a single node on the cluster.
+If strict packing is not feasible, bundles can be placed on multiple `PACK` nodes.
```

Comment:
I moved this text around. I'd like a technical review to make sure this move makes sense.